pattern-library/fonts/edx-icons/edx-icons.svg

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Looks like this is generated by IcoMoon, right? Any manual steps/changes you made from the produced file?